lms/djangoapps/edraak_bayt/views.py

```diff
@@ -27,6 +26,7 @@ def get_student_email(request):
     user_email = request.POST['bayt_email']
     course_name = request.POST['course_name']
     course_id = request.POST['course_id']
+    cert_end = request.POST['cert_end']
```

Comment:
I wouldn't trust the user on that. Although it a minimal issue. 

 - [ ] Let's calculate it here instead of passing it from another module.